backend/tests/Feature/Web/Operator/WoTimeLotPickingTest.php

```
@@ -0,0 +1,259 @@
+        $this->enableLotTracking();
+        $this->makeLot('LOT-A', 50);
+
+        $this->get("/operator/batch-step/{$this->step->id}/pick-preview")->assertRedirect();
+        $this->post("/operator/batch-step/{$this->step->id}/start")->assertRedirect();
+        $this->assertSame(0, AllocationLotPick::count());
+    }
+
+    public function test_wrong_role_is_forbidden(): void
+    {
+        $this->enableLotTracking();
+        $viewer = User::factory()->create();
```

PHPStan (2.2.2)

[error] Call to an undefined static method App\Models\User::factory().


(staticMethod.notFound)